### PR TITLE
Rename field `offset` to `log.offset`

### PR DESCRIPTION
**Pull request by ruflin**
* Update tests accordingly
* Remove checks from JSON tests as log prefix now used for log.offset
* Update generated files

### CODE REVIEW

File: filebeat/tests/system/test_registrar.py [hunk 1]
```diff
@@ -845,6 +845,7 @@ def test_clean_inactive(self):
         if os.name == "nt":
             assert data[0]["offset"] == 3
         else:
+            print data[0]
```
Comment (kvch) [inline filebeat/tests/system/test_registrar.py]: Leftover debug message?

File: filebeat/tests/system/test_registrar_upgrade.py [hunk 2]
```diff
@@ -78,6 +78,7 @@ def run_and_validate(self):
 
         data = self.get_registry()
         assert len(data) == 1
+        print data[0]
```
Comment (kvch) [inline filebeat/tests/system/test_registrar_upgrade.py]: Also leftover?

File: filebeat/tests/system/test_registrar.py (hunk 1 above)
Comment (ruflin) [reply filebeat/tests/system/test_registrar.py]: yes, removed.

File: filebeat/tests/system/test_registrar_upgrade.py (hunk 2 above)
Comment (ruflin) [reply filebeat/tests/system/test_registrar_upgrade.py]: yes, removed

**Pull request merged**